lib/ts/recipe/thirdpartypasswordless/types.ts

```diff
@@ -44,7 +49,7 @@ export type UserInput = {
             builder: OverrideableBuilder<RecipeInterface>
         ) => RecipeInterface;
     };
-};
+} & RecipeModuleUserInput<PreAndPostAPIHookAction>;
```

Comment:
How is this change related to this PR?

Reply:
Its not, theres another PR (ive linked it in the PR description) that should be merged before this one